website/content/blog/2024-03-14-meeting.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Board Meeting Minutes
+date: "2024-03-14"
+excerpt: Bounties, Bounties, Bounties, this is the core focus for the team, making big plans and projects for contributors to pick up and enable the Foundation to distribute the funds donated, as well as further accelerating documentation and samples.
+tags: foundation
+author: MonoGame Foundation
+image: /images/blog/cover/monogame_foundation.png
+ogImage: /images/blog/cover/monogame_foundation.png
+---
+
+## Attendee's
+
+- Dean Ellis - President
+- Dominique Louis - Treasurer
+- Simon Jackson - Corporate Secretary
+- Tom Spilman - Banking Matters
+- Marko Jeremic - Board Chairman
+- Thomas Altenburger - Board Member
+
+## Agenda
+
+- 3.8.2 release
+- Documentation update
+- Fixing macOS experience
+- Fixing font rendering regression that 3.8.1 introduced
+- Updating SDL2
+- Bounties
+
+## Outcomes
+
+Bounties, Bounties, Bounties, this is the core focus for the team, making big plans and projects for contributors to pick up and enable the Foundation to distribute the funds donated, as well as further accelerating documentation and samples.
+
+Here is a summary of the outcomes from March's meeting:
+
+- Talk to accelerate the update of the NuGet templates, as they are not functional for iOS/Android as .NET 6 templates. Templates to be refreshed for the 3.8.2 release.
+- Work is needed to improve the Mac build experience, efforts are underway to address the Mac build pipeline experience.
+- Issue being tracked for Mac build issues - macOS and Linux setup experience · [Issue #8124 · MonoGame/MonoGame (github.com)](https://github.com/MonoGame/MonoGame/issues/8124)
+- Only blocker for 3.8.2 release are the Mac  & Content pipeline updates.
+- Font Rendering issues also high priority for 3.8.2.
+- Considering updates SDL2 for 3.8.2, automation to be updated to improve the workflow.
+- NativeAoT bounty had no applicants, so this bounty will be awarded to Tom Spillman, on top of the new Native Framework.
+- The DX12 bounty has a single applicant, process underway to award them the bounty and sign NDA's so that work can commence.
+- Issue clean-up is progressing, feedback support from community required to help identify non-issues.
+- The Content Pipeline update is mostly complete, so no bounty required. Targeting 3.8.2.
```

Comment:
Is this right? I thought we had a bunch of Content Pipeline bounties in the works? cc @harry-cpp

Reply:
This is what was approved on the MonoGame Foundation meeting notes, which we always ask everyone to review following the meeting.

But if changes are needed, now is the time.